src/main/java/com/automation/school/ad_main/Collection/MainRetailor.java

```diff
@@ -0,0 +1,97 @@
+package com.automation.school.ad_main.Collection;
+
+import java.util.*;
+
+public class MainRetailor {
+    public static void main(String[] args) {
+        //Start
+        System.out.println("Hello to  create new oder: 1) enter user name 2) indecat type 3) enter item name" );
+        // New user registration
+        User user1 = new User();
+        Scanner myNewUserObj = new Scanner(System.in);  // Create a Scanner object
+        System.out.println("Hello please create new User");
+        String user = myNewUserObj.nextLine(); // Enter username and press Enter     /* create dynamic var
+        System.out.println("Username is: " + user);
+
+        // Create Adress
+        Scanner enterFullAddress1 = new Scanner(System.in);  // Create a Scanner object
```

Comment:
no need in another scanner.
you've already created one on line 11

Reply:
Duplicated  Scanner was deleted